tests/apps/test.management-app.jsx

```diff
@@ -60,9 +60,9 @@ describe('management/app', function() {
     var mgmt = TestUtils.findRenderedComponentWithType(
       view, FakeManagement
     );
-    // The first child should be the sign-in component.
+    // The second child should be the sign-in component.
     // Make sure it gets the access token.
-    var signIn = mgmt.props.children[0];
+    var signIn = mgmt.props.children[1];
```

Comment:
mmm, we should probably do this a better way, like, iterate the children and match on component type. Not for this patch though :)